Mobile/starter_project_matador/lib/features/example/presentation/screen/write_article_screen.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+
+class WriteArticlePage extends StatelessWidget {
```

Comment:
I would rename this to ArticleFormPage because it will also be used to edit articles.